src/libraries/System.Drawing.Primitives/src/System/Drawing/KnownColorNames.cs

```diff
@@ -149,6 +149,7 @@ internal static class KnownColorNames
             "Plum",
             "PowderBlue",
             "Purple",
+            "RebeccaPurple",
```

Comment:
Same scenario, I believe this must match the enum order. This case doesn't have a corresponding comment, however and likely should above the array.

Reply:
i will add the comment soon

Reply:
i added a comment. I hope its good enough
"//  Names of all colors (in order of definition in the KnownColor enum)."